src/Components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,25 @@
+import Button, { ButtonProps } from '@mui/material/Button'
```

Comment:
Redundant component, doesn't help at all, brings unnecessary complexity and bloats the codebase for no purpose :man_facepalming:.